### ISSUE TITLE
does it support concurrency

### ISSUE THREAD

**Issue by gek169** [alloca123/allocalang#1]
all the hot new languages have it does this one?

also wheres the thousand-line TOS banning hate speech?

**Comment by alloca123** [comment]
yes it supports concurrency. here are some libraries for concurrency: [openmp](https://www.openmp.org/), [pthreads](https://en.wikipedia.org/wiki/POSIX_Threads), [libdill](http://libdill.org/), [libmill](http://libmill.org/), [libaco](https://github.com/hnes/libaco), and many others.

Oh and thanks for the idea, now time to make a 2 thousand line CoC banning hate speech!

**Comment by gek169** [comment]
wheres the compiler for allocalang

**Comment by alloca123** [comment]
@gek169 allocalang.sh ~~btw allocalang was made as a dumb joke, don't take it seriously~~

**Comment by gek169** [comment]
oh yeah why you unadd me thats the next question
Im a javascript brogrammer now 
time to gain 200 pounds shave my head and grow a huge beard

**Comment by gek169** [comment]
keggek#3426

**Comment by alloca123** [comment]
@gek169 if you mean discord, i didn't unadd you personally, i just deleted my discord account. I've finally quit discord.

**Comment by alloca123** [comment]
> Im a javascript brogrammer now

Sorry to break it to you, but you are now required to buy a macbook, drink starbucks, and spend your free time canceling people on twitter.

**Comment by gek169** [comment]
aw fuck!!!!

**Comment by gek169** [comment]
> @gek169 if you mean discord, i didn't unadd you personally, i just deleted my discord account. I've finally quit discord.

That's a good thing, actually.
Stay in touch.
